openvino/cc/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set(TARGET_NAME cc)
+
+add_library(${TARGET_NAME} INTERFACE)
+
+add_library(openvino::cc ALIAS ${TARGET_NAME})
+
+target_link_libraries(${TARGET_NAME} INTERFACE openvino::itt)
+
+target_include_directories(${TARGET_NAME} INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
+
+if(${ENABLE_SUBSET} STREQUAL "ANALYZE")
+    target_compile_definitions(${TARGET_NAME} INTERFACE -DENABLE_SUBSET_FIND)
+elseif(${ENABLE_SUBSET} STREQUAL "ON")
+    target_compile_definitions(${TARGET_NAME} INTERFACE -DENABLE_SUBSET)
+    if(MSVC)
+        target_compile_options(${TARGET_NAME} INTERFACE /FI"${SUBSET_DEFS}")
+    else()
+        target_compile_options(${TARGET_NAME} INTERFACE -include "${SUBSET_DEFS}")
+    endif()
```

Comment:
can we introduce a cmake function `ie_force_include` and add it to a common place like `cmake/os_flags.cmake`?

Reply:
fixed